src/optimizer/plugins/RemoveGeneratedTextRulesOptimizer.ts

```diff
@@ -18,7 +20,14 @@ export default {
   description: 'Remove rules on "text" which represent generated content.',
 
   optimize(pkg: Package): void {
-    [...pkg.profiles, ...pkg.extensions, ...pkg.valueSets, ...pkg.codeSystems].forEach(resource => {
+    [
```

Comment:
I can't comment on the actual line, but I think we want to apply some other optimizers:
 * `remove_publisher_derived_date_rules` to at least `Logical`s as well. I'm not sure if `Resource`s can even make it through the publisher, so they probably don't matter so much.
 * `resolve_parent_urls` to both `Logical` and `Resource` I think.
 * `simplify_array_indexing` to both `Logical` and `Resource` for caret rules.